components/tooltip/tooltip.js

```diff
@@ -946,13 +947,31 @@ class Tooltip extends RtlMixin(LitElement) {
 	}
 
 	_updateShowing() {
-		this.showing = this._isFocusing || this._isHovering || this.forceShow || this.#isHoveringTooltip;
+		if (this._initiallyFocused === undefined) return;
+		this.showing = !this._initiallyFocused && (this._isFocusing || this._isHovering)
+			|| this.forceShow || this.#isHoveringTooltip;
 	}
 
 	_updateTarget() {
+		const newTarget = this._findTarget();
+		if (this._target === newTarget) {
+			return;
+		}
+
 		this._removeListeners();
-		this._target = this._findTarget();
+		this._target = newTarget;
+
 		if (this._target) {
+			const targetDisabled = this._target.hasAttribute('disabled') || this._target.getAttribute('aria-disabled') === 'true';
```

Comment:
Did you confirm with Jeff that we'd want to do this anywhere that this behaviour could happen? It seems like there would be potential for this to be something a tool is relying on, though maybe we'd want to broadly prevent that, and he'd have better insight into that.

Reply:
I'll run it by him and get back on this. 

It does seem unlikely that there would be an instance of an element getting a tooltip attached to it while focused/active/hovered && it's disabled where we would want the tooltip displayed though.

Reply:
Jeff felt comfortable with this approach - he couldn't think of any obvious scenarios where we would actively be looking to have the previous behaviour.

He did think that some soak time and monitoring would be good - and that it's not something that's necessarily easy to check for via code search since it'd be dynamic in nature